Adds a process for adding a plugin. (#50)
WordPress/data-liberation: submit-code/readme.md

# Submitting Code to the Data Liberation Project

> You have code that can users move their data freely on the web? Awesome. This readme will provide you with the details and requirements on how to submit your code. If you get stuck along the way, [open an issue](https://github.com/WordPress/data-liberation/issues/new) or reach out in the [#data-liberation slack channel](https://wordpress.slack.com/archives/C069AKUBPHB).

Reviewer: ```suggestion
You have code that can help users move their data freely on the web? Awesome! This readme provides the details and requirements for how to submit your code. If you get stuck along the way, [open an issue](https://github.com/WordPress/data-liberation/issues/new) or reach out in the [#data-liberation slack channel](https://wordpress.slack.com/archives/C069AKUBPHB).
```

# Requirements

You need to answer yes to the following items:

> - My code contains a `readme.md` in the root directory?

Reviewer: For consistency maybe drop the question mark, and replace with colon for the sub items
```suggestion
- My code contains a `readme.md` in the root directory:
```
